### ISSUE TITLE
I set the parameters through the README, trained the VOC2007 dataset, and the obtained model was tested but could not be detected.WHY？

### ISSUE THREAD

**Issue by zengbing1992** [balancap/SSD-Tensorflow#274]
The set parameters：
{'adadelta_rho': 0.95,
 'adagrad_initial_accumulator_value': 0.1,
 'adam_beta1': 0.9,
 'adam_beta2': 0.999,
 'batch_size': 16,
 'checkpoint_exclude_scopes': None,
 'checkpoint_model_scope': 'ssd_300_vgg',
 'checkpoint_path': './checkpoints/ssd_300_vgg.ckpt/ssd_300_vgg.ckpt',
 'clone_on_cpu': False,
 'dataset_dir': './tfrecords_voc',
 'dataset_name': 'pascalvoc_2007',
 'dataset_split_name': 'train',
 'end_learning_rate': 1e-06,
 'ftrl_initial_accumulator_value': 0.1,
 'ftrl_l1': 0.0,
 'ftrl_l2': 0.0,
 'ftrl_learning_rate_power': -0.5,
 'gpu_memory_fraction': 0.8,
 'ignore_missing_vars': False,
 'label_smoothing': 0.0,
 'labels_offset': 0,
 'learning_rate': 0.0001,
 'learning_rate_decay_factor': 0.94,
 'learning_rate_decay_type': 'exponential',
 'log_every_n_steps': 10,
 'loss_alpha': 1.0,
 'match_threshold': 0.5,
 'max_number_of_steps': 120000,
 'model_name': 'ssd_300_vgg',
 'momentum': 0.9,
 'moving_average_decay': None,
 'negative_ratio': 3.0,
 'num_classes': 21,
 'num_clones': 1,
 'num_epochs_per_decay': 2.0,
 'num_preprocessing_threads': 4,
 'num_readers': 4,
 'opt_epsilon': 1.0,
 'optimizer': 'adam',
 'preprocessing_name': None,
 'rmsprop_decay': 0.9,
 'rmsprop_momentum': 0.9,
 'save_interval_secs': 600,
 'save_summaries_secs': 60,
 'train_dir': './logs2/',
 'train_image_size': None,
 'trainable_scopes': None,
 'weight_decay': 4e-05}
{'anchor_offset': 0.5,

Why is the trained model completely useless? If the training loss is around 0.0504, there is no way to drop it. Is it necessary to adjust other parameters? Thank you.
@balancap

**Comment by Breezeqiuyu** [comment]
Have you solved it now?How did you do it?
